src/util/Util.h

```diff
@@ -16,47 +16,42 @@
 #include <gtk/gtk.h>
 
 #include <string>
-using std::string;
-
 #include <functional>
+#include <limits>
 
+using std::string;
 
-class Util
+namespace Util
```

Comment:
if this is a namespace, please name it `util`

Reply:
Normally I would do that and for project independent stuff I do it even in this project. But the naming convention in the Wiki says: "Names are CamelCase", since Util:: is a project dependant namespace I'll name it with CamelCase; It also does not make a different.